Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'rspec', '~> 3.0'
 gem 'rack-test', '~> 0.6.3'
 gem 'sqlite3', '~> 1.3.11'
 gem 'rails', "~> #{ENV["RAILS_VERSION"] || '4.2.5'}"
+gem 'minitest', '~> 5.8.0'
```

Comment:
test::unit is default testing framework for rails versions < 4.0, so tests were failing on those versions.  Is adding it to the Gemfile best way to handle this?